samples/XCT.Sample.WPF/DrawingViewRenderer.wpf.cs

```diff
@@ -53,55 +55,96 @@ protected override void OnElementChanged(ElementChangedEventArgs<DrawingView> e)
 			if (e.OldElement != null)
 			{
 				canvas!.Strokes.StrokesChanged -= OnStrokesChanged;
-				Element!.Points.CollectionChanged -= OnCollectionChanged;
+				Element!.Lines.CollectionChanged -= OnCollectionChanged;
 				if (Control != null)
 					Control.PreviewMouseDown -= OnPreviewMouseDown;
 			}
 		}
 
-		void OnCollectionChanged(object sender, NotifyCollectionChangedEventArgs args) => LoadPoints();
+		void OnCollectionChanged(object sender, NotifyCollectionChangedEventArgs args)
+		{
+			canvas!.Strokes.StrokesChanged -= OnStrokesChanged;
+			canvas.Strokes.Clear();
+			LoadLines();
+			canvas.Strokes.StrokesChanged += OnStrokesChanged;
+		}
+
+		void OnPreviewMouseDown(object sender, MouseButtonEventArgs e) => Clear();
 
-		void OnPreviewMouseDown(object sender, MouseButtonEventArgs e)
+		void Clear(bool force = false)
 		{
-			canvas!.Strokes.Clear();
-			Element.Points.Clear();
+			if (!Element.MultiLineMode || force)
+			{
+				canvas!.Strokes.Clear();
+				Element.Lines.Clear();
+			}
 		}
 
 		void OnStrokesChanged(object sender, StrokeCollectionChangedEventArgs e)
 		{
-			Element.Points.CollectionChanged -= OnCollectionChanged;
+			Element.Lines.CollectionChanged -= OnCollectionChanged;
```

Comment:
Maybe I didn’t understand your question, but I will try to answer.
DrawingView doesn’t contain collection of lines any more. Instead we added an additional layer - Line. Almost all DrawingView properties are moved to the Line class. 
I don’t think it will be correct to leave the name Points. Furthermore it is a Collection of Lines, not Points.

Reply:
You're right @pictos and not even ABI, API as well :)

That's why I was wondering what we should do with this one. Because this means in 1.2 we have the implementation that we have right now and in 1.3 we have a whole bunch of breaking changes with the DrawingView already while we actually said that we should deprecate something first and then actually remove it.

I think this is a major improvement though and I think if we communicate this in the release notes very clear then it's not too much of a problem?